### ISSUE TITLE
propose to move binary build scripts to new repo

### ISSUE THREAD

**Issue by ipelupessy** [amusecode/amuse#571]
I think the scripts and stuff needed to build binary releases can move to their own repository - they are not related to amuse per se

**Comment by ipelupessy** [comment]
there are more things that can be moved; the old svn repo functioned more like the amuse project page, so it makes sense to move them higher up

**Comment by ipelupessy** [comment]
note that you can use filtering where you want to retain history on the files...

**Comment by rieder** [comment]
Yes, I'd be in favour of a bit of housekeeping on the main AMUSE repository.

**Comment by ipelupessy** [comment]
house keeping still necessary

**Comment by ipelupessy** [comment]
todo:
 - [ ] copy (filter) support/distribute to new repo amuse-binbuild
 - [ ] remove support/distribute from repo
 - [ ] optionally fix the newly created binbuild repo (it will not work anymore)
 - [ ] delete(?) support/macports
 - [ ] check buildbot and all test related stuff in support
 - [ ] check remaining other stuff in support
